### ISSUE TITLE
The version of CheXpert

### ISSUE THREAD

**Issue by SZUHvern** [RyanWangZf/MedCLIP#11]
Thank you for sharing the code!

Since CheXpert V1 has two released versions: [CheXpert-v1.0 Original (~ 439 g)] and [CheXpert - v1.0 Downsampled (~ 11 g)], I would like to know which one do you choose? Also, have you compared the difference of performance between them or have any suggestion about them?

Waiting for your kindly reply.
Many thanks.

**Comment by RyanWangZf** [comment]
Hi, we used the Kaggle downsampled 12G version. We didn't try the big one.

**Comment by icannistraci** [comment]
Hi @RyanWangZf, I was wondering how you were able to generate the chexpert-5x200 dataset when using only the downsample version. As far as I understand, the validation split is too small to generate a dataset of 1k images and can only be created starting from the training split or the complete chexpert dataset (439gb). Thank you in advance for your help!

**Comment by RyanWangZf** [comment]
Hi, we follow the settings mentioned in Sec 4.1 in [1], where a subset of training data is held out and used as test set.

[1] Huang, S. C., Shen, L., Lungren, M. P., & Yeung, S. (2021). Gloria: A multimodal global-local representation learning framework for label-efficient medical image recognition. In Proceedings of the IEEE/CVF International Conference on Computer Vision (pp. 3942-3951).

**Comment by lorenzoscottb** [comment]
Hi and thanks for sharing the code.

Would you be able to share the obtained CheXpert 5x200 dataset, or the code used to sample it? 

It would be great for some replication/testing experiments.

Many thanks.
